lib/pronto/golang/tools/errcheck.rb

```diff
@@ -16,7 +16,7 @@ def installed?
       def parse_line(line)
         file_path, line_number, _, _ = line.split(':')
 
-        return File.expand_path(file_path), line_number, :warning, message
+        [file_path, line_number, :warning, message]
```

Comment:
Is there any particular reason you removed the explicit `return`?

Reply:
Ruby returns last statement by design. I though that this way will be cleaner and more rubbish.

Reply:
Yes, ruby automatically returns the last statements. However that does not mean that it is necessarily better, as it is less explicit.
I generally prefer explicit returns even if they are at the end of a function.